tests/schema/types/test_datetime.py

```diff
@@ -105,3 +107,66 @@ def deserialize(self, arg: typing) -> bool:
 
     assert not result.errors
     assert Query.deserialized == instance
+
+
+def execute_mutation(value):
+    @strawberry.type
+    class Query:
+        ok: bool
+
+    @strawberry.type
+    class Mutation:
+        @strawberry.mutation
+        def datetime_input(
+            self, datetime_input: datetime.datetime
+        ) -> datetime.datetime:
+            assert isinstance(datetime_input, datetime.datetime)
+            return datetime_input
+
+    schema = strawberry.Schema(query=Query, mutation=Mutation)
+
+    return schema.execute_sync(
+        f"""
```

Comment:
Instead of an f-string a GraphQL variable could be used here. This is just a personal preference, but would also make semantic sense. Here is an example from another test:

https://github.com/strawberry-graphql/strawberry/blob/e6b005ce98df53f923c477fc11ab69c0a8f58736/tests/schema/test_custom_scalar.py#L50-L54